plugins/Denizen/scripts/cooking/allergies.dsc

```diff
@@ -0,0 +1,77 @@
+allergies:
+    type: world
+    debug: false
+    events:
+        on player dies:
+            - if <util.random_chance[12]>:
+                - define allergy <script[allergy_info].data_key[valid_allergies].keys.random>
+                - define effect <script[allergy_info].data_key[valid_effects].keys.random>
+                - flag player allergic:<[allergy]>
+                - flag player allergy_effect:<[effect]>
+                - stop
+            - flag player allergic:!
+            - flag player allergy_effect:!
+
+        on player consumes item flagged:allergic:
+
+            - define allergies <script[allergy_info].data_key[valid_allergies.<player.flag[allergic]>]>
+            - if <[allergies].contains[<context.item.script.name.if_null[<context.item.material.name>]>]>:
+                - foreach <script[allergy_info].data_key[valid_effects.<player.flag[allergy_effect]>]> as:a key:b:
+                    - cast <[b]> duration:<[a].get[duration]> amplifier:<[a].get[amplifier]> hide_particles no_icon
```

Comment:
```suggestion
                    - cast <[b]> duration:<[a].get[duration]> amplifier:<[a].get[amplifier].sub[1]> hide_particles no_icon
```
Amplification starts at zero, therefore all the amplifiers should either be reduced by one in the data script, or they should be reduced by one here. This is assuming that every time you put an amplifier you meant the direct potion strength, not Denizen's system.

Reply:
i already tested the potency of the effects with the current values so

Reply:
check again nerd. cast youself with wither amplifier level 1 and see what happens

Reply:
no like i meant the desired results w the potion effects like how fast you take damage etc

Reply:
...what

Reply:
idk ill jst change the values itself